### PR TITLE
Grid Row Selection

### PR DESCRIPTION
**Pull request by agubler**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [x] Unit tests are included in the PR

**Description:**

Support for row selection within the grid, returns an array of selected rows via the `onRowSelect` property. Supports multiple row selection when using either the `cmd` or `ctrl` key.

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-pmw3yjy31.now.sh
* Tests Deployment: https://dojowidgets-p86puxxlx.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-bunxjtot1.now.sh
* Tests Deployment: https://dojowidgets-fvkec5xnx.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-ej3liz6va.now.sh
* Tests Deployment: https://dojowidgets-amlx8k0ku.now.sh

File: src/theme/default/grid-row.m.css [hunk 1]
```diff
@@ -4,3 +4,7 @@
 	height: 30px;
 	border-left-style: none;
 }
+
+.selected {
```
Comment (tomdye) [inline src/theme/default/grid-row.m.css]: should this have a comment?

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-7m1fh91iy.now.sh
* Tests Deployment: https://dojowidgets-lso7uxuop.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-7bdvvxpe3.now.sh
* Tests Deployment: https://dojowidgets-ja6apcjwv.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-5zisfim78.now.sh
* Tests Deployment: https://dojowidgets-3sfy289mt.now.sh

File: src/theme/default/grid-row.m.css (hunk 1 above)
Comment (agubler) [reply src/theme/default/grid-row.m.css]: @tomdye done

**Comment by agubler** [comment]
@tomdye could I get another pass of this please?

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-ny0tilpru.now.sh
* Tests Deployment: https://dojowidgets-ca3ani9x1.now.sh

**Comment by agubler** [comment]
All the tests are passing with node and the failures are due to incompatible chrome driver versions only, merging.

**Pull request merged**